workflow-to-js-transformer/README.md

```diff
@@ -71,40 +71,65 @@ Which given a projectId of "myProjectId" would be output after transformation as
 }
 ```
 
-*tenantId*
+*taskSettings*
 
-A source of tenantId may be specified to indicate that a value for a property should come from a `tenantId` parameter that will
- be available during workflow transformation.
-
-```
-"customData": {
-     "myProperty" : { "source" : "tenantId"}
- }
-```
-
-Which given a projectId of "1234" would be output after transformation as;
+A source of taskSettings maybe specified to indicate that a the key specified can be used to look up the correct value on the input message's customdata. If the key is accompanied by a defaultValue then if the value is not present the default value will be used.
```

Comment:
```suggestion
A source of taskSettings may be specified to indicate that a value for a property should come from the input message's customdata. If defaultValue is supplied this will be used when a value is not present in the custom data.
```

https://blogs.transparent.com/english/maybe-versus-may-be/